ui/src/Utils/userIdValidator.ts

```diff
@@ -0,0 +1,36 @@
+import {Option} from '../CommonTypes/Option';
+
+export interface ValidateUserResult {
+    options: Option[];
+    notValid: string;
+}
+
+export const emptyValidateUserResult = Object.freeze({
+    options: [],
+    notValid: '',
+});
+
+export function makeOption(name: string): Option {
+    return {value: name, label: name};
+}
+
+export const nameSplitPattern = new RegExp(/,|;|\s/);
+export const userIdPattern = new RegExp(/^[a-zA-Z][a-zA-Z0-9]{1,8}$/);
```

Comment:
I've seen this regex in a couple of places now - can we consolidate into one place?

Reply:
To do: Add card to move cdsid (user id) validation out to an environment variable to make code more open source friendly, for other organizations that don't use cdsids.